hack/hypernetes/ceph.sh

```diff
@@ -20,10 +20,24 @@ set -o pipefail
 
 function kube::util::setup_ceph() {
 	echo "Start $FUNCNAME"
+	rm -rf /etc/ceph
+	mkdir -p /etc/ceph
+	if ! kube::util::do_setup_ceph ; then
+		echo "Preinstall some package before exectue ceph-deploy..."
+		kube::util::ceph_preinstall
+		kube::util::do_setup_ceph
```

Comment:
It's just a workaround here. Actually these two approach will install `ceph` from different repo, `kube::util::ceph_preinstall` will install from centos repo, ie: mirrors from aliyun, while `do_setup_ceph` will install `ceph` from official repo through `ceph-deploy`.  The second one is very slow and may always fail in somewhere like China.